src/test/deploy/functions/params.spec.ts

```diff
@@ -80,12 +98,12 @@ describe("resolveParams", () => {
         name: "foo",
         default: "bar",
         type: "string",
-        input: { type: "text", text: {} },
```

Comment:
Isn't this the wrong field we deleted?

Reply:
Technically, it isn't--the type: field is mandatory until #4874 goes in, since I separated out the 10 SWE-hour banana into a different PR. Whichever one goes second is just going to have to fix merge conflicts, but it should be trivial.